### DISCUSSION TITLE
laravel-sends: Keep track of outgoing emails

### DISCUSSION THREAD

**Discussion by stefanzweifel** [HassanZahirnia/laravel-package-ocean#69] [Package Suggestions]
A package to keep track of outgoing emails in your Laravel application. 

- https://github.com/stefanzweifel/laravel-sends
- https://packagist.org/packages/wnx/laravel-sends

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package! It will be live in a moment.

**Reply by stefanzweifel** [reply]
Thanks Hassan!
